index.html

```diff
@@ -79,15 +77,17 @@
             <p class="connect">LET’S CONNECT</p>
           </div>
           <div class="media-icons">
-              <a href=""><img src="./images/twitter.png" alt=""></a>
-              <a href=""><img src="./images/Linkedin.png" alt=""></a>
-               <a href=""><img src="./images/mail.png" alt=""></a>
-               <a href=""><img src="./images/github.png" alt=""></a>
-              <a href=""><img src="./images/peace.png" alt=""></a>
+            <ul>
+              <li><a href=""><img src="./images/twitter.png" alt=""></a></li>
+              <li><a href=""><img src="./images/Linkedin.png" alt=""></a></li>
+              <li><a href=""><img src="./images/mail.png" alt=""></a></li>
+              <li><a href=""><img src="./images/github.png" alt=""></a></li>
+              <li><a href=""><img src="./images/peace.png" alt=""></a></li>
+            </ul>
           </div>
-    </section>
-    <article class="workcards">
-      <section class="workcard">
+        </div>
+    <div class="workcards">
+      <article class="workcard">
         <img src="./images/Snapshoot.png" alt="">
```

Comment:
- [ ] As the previous reviewer mentioned, please know that `article` should be used when you have repetitive code, here it's advised to use `main` semantic tag. 😁